notebooks/data-sources/oc-github-repo/github_PR_EDA.ipynb

```diff
@@ -2,17 +2,19 @@
  "cells": [
```

Comment:
<p>For flow, it might make sense to swap order of "New DataFrame" and "Time to Merge" prose</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/293/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>thanks for pointing that out @isabelizimm. I think it's safe to remove the "New Dataframe" section altogether from here since we make modifications in the same pr_df dataframe</p>